src/insert.rs

```diff
@@ -227,7 +204,12 @@ impl<T> Insert<T> {
         // So, instead we control it manually here and rely on the system allocator.
         let chunk = self.take_and_prepare_chunk()?;
 
-        let sender = self.sender.as_mut().unwrap(); // checked above
+        // checked above
```

Comment:
this comment is related only to `unwrap()`, not for the whole statement

Reply:
However, it's the wrong comment now because "above" only the existence of `self.request` is checked.

Reply:
We can make code more clear by replacing `Option<InsertRequest>` with
```rust
enum InsertRequest {
    NotStarted {
        client: Client,
        sql: String,
    },
    Active {
        sender: ChunkSender,
        handle: JoinHandle<Result<()>>,
    },
    Terminated {
        handle: JoinHandle<Result<()>>,
    },
    Aborted,
}
```
and some helpers `InsertRequest::handle() -> Option<_>`, `InsertRequest::sender() -> Option<_>`